### PR TITLE
Change warning check version Java 16

### PR DESCRIPTION
**Pull request by Euphillya**
Mojang use now Java 16 for 1.17+ 
https://www.minecraft.net/en-us/article/minecraft-snapshot-21w19a

### CODE REVIEW

File: Spigot-Server-Patches/0609-Add-warning-for-servers-not-running-on-Java-16.patch [hunk 1]
```diff
@@ -38,15 +40,15 @@ index 0000000000000000000000000000000000000000..c6ea429819c07e7f4bc257cad73463a0
...
 +            final Logger logger = LogManager.getLogger();
 +            logger.warn("************************************************************");
 +            logger.warn("* WARNING - YOU ARE RUNNING AN OUTDATED VERSION OF JAVA.");
 +            logger.warn("* PAPER WILL STOP BEING COMPATIBLE WITH THIS VERSION OF");
 +            logger.warn("* JAVA WHEN MINECRAFT 1.17 IS RELEASED.");
 +            logger.warn("*");
 +            logger.warn("* Please update the version of Java you use to run Paper");
-+            logger.warn("* to at least Java 11. When Paper for Minecraft 1.17 is");
++            logger.warn("* to at least Java 16. When Paper for Minecraft 1.17 is");
 +            logger.warn("* released support for versions of Java before 11 will");
```
Comment (kennytv) [inline Spigot-Server-Patches/0609-Add-warning-for-servers-not-running-on-Java-16.patch]:
```suggestion
+            logger.warn("* released support for versions of Java before 16 will");
```

File: Spigot-Server-Patches/0609-Add-warning-for-servers-not-running-on-Java-16.patch [hunk 2]
```diff
@@ -1,7 +1,9 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Kyle Wood <demonwav@gmail.com>
 Date: Wed, 2 Dec 2020 21:58:45 -0800
-Subject: [PATCH] Add warning for servers not running on Java 11
+Subject: [PATCH] Add warning for servers not running on Java 16
+
+Mojang Informations https://www.minecraft.net/en-us/article/minecraft-snapshot-21w19a
```
Comment (kennytv) [inline Spigot-Server-Patches/0609-Add-warning-for-servers-not-running-on-Java-16.patch]:
```suggestion
Mojang announcement https://www.minecraft.net/en-us/article/minecraft-snapshot-21w19a
```

**Review by kennytv — CHANGES_REQUESTED** [review]
Please don't use the web editor for this, even if it's such simple one-line changes.

**Comment by kennytv** [comment]
Keeping the link as is would be confusing as well.

**Comment by Proximyst** [comment]
4efe8b498b0e298d727c6dbd31ccc38f60cf0655

**Pull request closed without merging**